README.md

```diff
@@ -56,8 +61,6 @@ function NewComponent(props) {
 }
 ```
 
-### Storybook
-
-There should be no need to run the static Storybook instances. If the static Storybook npm script is run, make sure to not commit the `storybook/storybook-static` directory.
```

Comment:
I think this note about deleting the `storybook-static` directory if you ever run static, should stay.